src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -147,6 +147,13 @@ public String removeUserFromTeam(String teamId) {
     //-------------------------------------
     // EXPERIMENTS
     //-------------------------------------
+    public String getExperiment(String id) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + expEndpoint + "/" + id;
+    }
+
+    public String getExperimentDetails(String teamId, String expId) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + expEndpoint + "/teams/" + teamId + "/experiments/" + expId + "/experimentDetails";
+    }
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "/experiments/" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)